Jenkinsfile.public

```diff
@@ -99,6 +99,36 @@ pipeline {
                     }
                 }
 
+                stage('npm') {
+                    steps {
+                        dir("web/satellite") {
+                            sh 'npm run build'
+                            sh 'npm run lint'
+                            // TODO: reenable with vue4
+                            sh script: 'npm audit', returnStatus: true
+                            sh 'npm run test'
+                        }
+                    }
+                }
+            }
+        }
+
+        stage('Verification 2') {
+            parallel {
+                stage('Tests [gomobile]') {
+                    environment {
+                        // docker container IP
+                        STORJ_NETWORK_HOST4 = '172.17.0.2'
+                        STORJ_NETWORK_HOST6 = '172.17.0.2'
```

Comment:
Lets not use "non localhost" addresses.

Reply:
Problem is that localhost address (with `storj-sim`) is not accessible for Android emulator but I just found that maybe there is a workaround for that. I will try to remove this.